src/controllers/movies.js

```diff
@@ -0,0 +1,81 @@
+
+const prisma = require('../utils/prisma')
+
+const getAllMovies = async(req, res) => {
+    return await prisma.movie.findMany({
+        select: {
+            id: true,
+            title: true,
+            runtimeMins: true,
+            createdAt: true,
+            updatedAt: true,
+            screenings: true
+        }
+    })
+}
+
+const createMovie = async(req, res) => {
+    
+    const {title, runtimeMins} = req.body
```

Comment:
I wish I had thought of doing this way in my code, instead of typing every time 'title: body.title'. Will think about it next time.